recipes/r-coop/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set version = '0.6-1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-coop
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/coop_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/coop/coop_{{ version }}.tar.gz
+  sha256: b7013bfdc4c9c1e2777c455bf527ec1a295c2ee166d6f4b3e5976a8a0f4727c3
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: memuse
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{native}}toolchain        # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}sed               # [win]
+    - {{posix}}grep              # [win]
+    - {{posix}}autoconf
+    - {{posix}}automake          # [not win]
+    - {{posix}}automake-wrapper  # [win]
+    - {{posix}}pkg-config
+    - {{posix}}make
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+
+  host:
+    - r-base
+
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('coop')"           # [not win]
+    - "\"%R%\" -e \"library('coop')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://github.com/wrathematics/coop
+  license: BSD
+  summary: 'Fast implementations of the co-operations: covariance, correlation, and cosine similarity.  The
+    implementations are fast and memory-efficient and their use is resolved automatically
+    based on the input data, handled by R''s S3 methods.  Full descriptions of the algorithms
+    and benchmarks are available in the package vignettes.'
+
+  license_family: BSD
+
+extra:
+  recipe-maintainers:
+  - mtekman
```

Comment:
have you used the conda_skeleton_helper script? You can add a few more maintainers here.

Reply:
Is it okay if I add you?

Reply:
https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml those can be added.